staff/cristian-astudillo/precourse/Pasapalabra/index.js

```diff
@@ -0,0 +1,197 @@
+    { letter: "q", answer: "queso", status: 0, question: "CON LA Q. Producto obtenido por la maduración de la cuajada de la leche" },
+    { letter: "r", answer: "raton", status: 0, question: "CON LA R. Roedor" },
+    { letter: "s", answer: "stackoverflow", status: 0, question: "CON LA S. Comunidad salvadora de todo desarrollador informático" },
+    { letter: "t", answer: "terminator", status: 0, question: "CON LA T. Película del director James Cameron que consolidó a Arnold Schwarzenegger como actor en 1984" },
+    { letter: "u", answer: "unamuno", status: 0, question: "CON LA U. Escritor y filósofo español de la generación del 98 autor del libro 'Niebla' en 1914" },
+    { letter: "v", answer: "vikingos", status: 0, question: "CON LA V. Nombre dado a los miembros de los pueblos nórdicos originarios de Escandinavia, famosos por sus incursiones y pillajes en Europa" },
+    { letter: "w", answer: "sandwich", status: 0, question: "CONTIENE LA W. Emparedado hecho con dos rebanadas de pan entre las cuales se coloca jamón y queso" },
+    { letter: "x", answer: "botox", status: 0, question: "CONTIENE LA X. Toxina bacteriana utilizada en cirujía estética" },
+    { letter: "y", answer: "peyote", status: 0, question: "CONTIENE LA Y. Pequeño cáctus conocido por sus alcaloides psicoactivos utilizado de forma ritual y medicinal por indígenas americanos" },
+    { letter: "z", answer: "zen", status: 0, question: "CON LA Z. Escuela de budismo que busca la experiencia de la sabiduría más allá del discurso racional" },
+]
+
+
+
+//VACIAR CONTADOR SI SE DESEA VOLVER A JUGAR.
+function vaciar() {
+    aciertos = 0,
+        fallos = 0;
+};
+
+//CONTADORES ACIERTOS Y FALLOS.
+let aciertos = 0;
+let fallos = 0;
+
+function juego() {
+    //INSERTAR NOMBRE JUGADOR
+    alert("Bienvenidos a Pasapalabra!!");
+    var nombre = prompt("Por favor, dinos tu nombre para poder participar.").toUpperCase();
+    if (nombre === '') {
+        return alert('Adiós!!');
+    };
+
+    alert(" Bienvenido " + nombre);
+    alert("Las reglas del juego son las siguientes : la pantalla lanzará la definición de una "
+        + "palabra y deberás adivinar qué palabra estamos tratando; si aciertas, es 1 punto; si fallas, es respuesta errónea, "
+        + "y si no sabes de momento, di 'pasapalabra' y se acumulará a la siguiente ronda hasta que contestes todas las definiciones.");
+    alert("De lo contrario, si no contestas o cancelas, el juego se termina de manera abrupta.");
+    alert('A continuacíon, daremos comienzo al Pasapalabra....')
+    alert('... Y empezamos!!!');
+
+    //COMIENZA EL JUEGO
+    for (const preguntas of questions) {
+        alert(preguntas.question)
+        let palabra = prompt('diga la palabra correspondiente... Si no lo sabe, diga "pasapalabra":').toLowerCase();
```

Comment:
can print ´preguntas.question´ inside the prompt, so you can see the question at the time you type the answer